tests/projects/dashboard_components/test_views_project_moderators.py

```diff
@@ -29,3 +32,146 @@ def test_edit_view(client, phase_factory):
+    assert len(mail.outbox) == 2
+
+
+@pytest.mark.django_db
+def test_initiator_can_delete_invitation(client, phase_factory):
+    phase, module, project, idea = setup_phase(
+        phase_factory, None, CollectFeedbackPhase)
+    url = component.get_base_url(project)
+    initiator = module.project.organisation.initiators.first()
+    client.login(username=initiator.email, password='password')
+    response = client.get(url)
+    assert_template_response(
+        response, 'meinberlin_projects/project_moderators.html')
+
+    data = {
+        'add_users': 'test1@foo.bar,test2@foo.bar',
+    }
+    response = client.post(url, data)
+    assert redirect_target(response) == \
+        'dashboard-{}-edit'.format(component.identifier)
+    assert ModeratorInvite.objects.get(email='test1@foo.bar')
+    invite_pk = int(ModeratorInvite.objects.get(email='test1@foo.bar').pk)
+    data = {
+        'submit_action': 'remove_invite',
+        'invite_pk': invite_pk
+
+    }
+    response = client.post(url, data)
+    assert redirect_target(response) == \
+        'dashboard-{}-edit'.format(component.identifier)
+    assert response.status_code == 302
+    assert not ModeratorInvite.objects.filter(email='test1@foo.bar').exists()
+    response = client.post(url, data)
+    assert redirect_target(response) == \
+        'dashboard-{}-edit'.format(component.identifier)
+    assert response.status_code == 302
+
+
+@pytest.mark.django_db
+def test_initiator_can_delete_moderator(client, phase_factory, user):
+    phase, module, project, idea = setup_phase(
+        phase_factory, None, CollectFeedbackPhase)
+    assert user not in module.project.moderators.all()
+    module.project.moderators.add(user)
+    url = component.get_base_url(project)
+    initiator = module.project.organisation.initiators.first()
+    client.login(username=initiator.email, password='password')
+    user_pk = user.pk
+    data = {
+        'submit_action': 'remove_user',
+        'user_pk': user_pk
+
+    }
+    response = client.post(url, data)
+    assert redirect_target(response) == \
+        'dashboard-{}-edit'.format(component.identifier)
+    assert response.status_code == 302
+    assert user not in module.project.moderators.all()
+    response = client.post(url, data)
+    assert redirect_target(response) == \
+        'dashboard-{}-edit'.format(component.identifier)
+    assert response.status_code == 302
+
+
+@pytest.mark.django_db
+def test_moderator_can_only_be_invited_once(
+        client, phase_factory, user):
+    phase, module, project, idea = setup_phase(
+        phase_factory, None, CollectFeedbackPhase)
+    assert user not in module.project.moderators.all()
+    module.project.moderators.add(user)
+    url = component.get_base_url(project)
+    initiator = module.project.organisation.initiators.first()
+    client.login(username=initiator.email, password='password')
+    with translation.override('en_GB'):
```

Comment:
@Rineee I found the translation stuff! https://docs.djangoproject.com/en/3.1/topics/testing/tools/#setting-the-language

Reply:
Oh cool! That rings a bell now, I used that for the email languages I think..not for testing though. Nice, so now we will hopefully remember it :)